app/styles/utilities.css

```
@@ -12,6 +12,12 @@
   margin: 0 auto;
 }
 
+.popover {
+  border: 1px solid var(--color-gray-3);
```

```suggestion
  border: 1px solid var(--border-gray);
```
See https://github.com/webkom/lego-webapp/pull/3518